test/ssltest_old.c

```diff
@@ -107,7 +103,7 @@ static unsigned int psk_server_callback(SSL *ssl, const char *identity,
                                         unsigned int max_psk_len);
 #endif
 
-static BIO *bio_err = NULL;
+BIO *bio_err = NULL;
```

Comment:
Why?

Reply:
Because this file meanwhile includes `testutil.h` which contains
```
extern BIO *bio_err;
```
so the `static` was flagged as inconsistent.

Reply:
BTW, I also had to clean up that file as well as `80-test_ssl_old.t` wr.t. `no-dh`.

Reply:
You can make this static again :wink:

Reply:
After removing `#include "testutil.h` from `ssltest_old.c` 
I had to revert the removal of `static` from `BIO *bio_err = NULL;`.